pkg/controller/controller_binding.go

```diff
@@ -320,6 +318,11 @@ func (c *controller) reconcileServiceInstanceCredential(binding *v1alpha1.Servic
 			c.recorder.Event(binding, api.EventTypeWarning, errorInjectingBindResultReason, s)
 			return err
 		}
+
+		// Create/Update for InstanceCredential has completed successful, so set Status.ReconciledGeneration to the
```

Comment:
Can you wrap this comment at 80 columns?

Reply:
Update is not applicable here - how about 'The bind operation completed successfully"